### PR TITLE
fixed ai issue

### LINKED ISSUE #32: **HIGH PRIORITY** BUG: FIX AI TAGLINE generation
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What went down?

image

Fix the code to make this work

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSoC'24 Extd Contributor
- [X] I'm a Hacktoberfest'24 registered Contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by Sourabh782**
Fixes #32

### CODE REVIEW

**Comment by Sourabh782** [comment]
@vansh-codes kindly review it

File: package-lock.json [hunk 1]
```diff
@@ -243,12 +244,13 @@
       }
     },
     "node_modules/@next/swc-darwin-arm64": {
```
Comment (vansh-codes) [inline package-lock.json:246]: Remove all this unwanted dependencies from PR, it was for your system and it shouldnt be required to everyone

**Review by vansh-codes — CHANGES_REQUESTED** [review]
Update the package-lock.json and remove unwanted dependencies

File: package-lock.json (hunk 1 above)
Comment (Sourabh782) [reply package-lock.json:246]: @vansh-codes can you specify which ones?

Comment (Sourabh782) [reply package-lock.json:246]: the nextjs version one?

Comment (vansh-codes) [reply package-lock.json]: swc-darwin-arm64
linux
node_modules/@next/swc-linux-arm64-gnu
msvc
and many more such linux and darwins dependencies

I guess you use linux thats why it was needed for you, but ig its not needed in the project, right?

Comment (vansh-codes) [reply package-lock.json]: Not the nextjs one, the other ones in package-lock.json

Comment (Sourabh782) [reply package-lock.json]: @vansh-codes i copied you current files, the only diff were of nextjs versions

Comment (vansh-codes) [reply package-lock.json]: @Sourabh782 I am talking about `package-lock.json` and not `package.json`

Comment (Sourabh782) [reply package-lock.json]: yup copied both

Comment (vansh-codes) [reply package-lock.json]: image

Comment (Sourabh782) [reply package-lock.json]: is it still showing this? i actually just copied your package-lock.json and make a commit

**Review by vansh-codes — APPROVED** [review]
Done ✅

**Comment by Sourabh782** [comment]
@vansh-codes this issue was marked level 2, can you fix it in pr?

**Comment by vansh-codes** [comment]
Acccording to the changes, it gets level1, i thought it would be complex thats why I mentioned level2 on the issue but its only 40 lines change, cant give level2
But good job!

**Comment by Sourabh782** [comment]
sir you can see changes in files, fixing routes, and solving error, also as it was marked level 2 initially.

**Comment by Sourabh782** [comment]
Thanks @vansh-codes sir :)) Really appreciated

**Pull request merged**